cvat/apps/events/serializers.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from dateutil import parser as datetime_parser
+import datetime
+import json
+
+from rest_framework import serializers
+
+class EventSerializer(serializers.Serializer):
+    scope = serializers.CharField(required=True)
+    obj_name = serializers.CharField(required=False, allow_null=True)
+    obj_id = serializers.IntegerField(required=False, allow_null=True)
+    obj_val = serializers.CharField(required=False, allow_null=True)
+    source = serializers.CharField(required=False, allow_null=True)
+    timestamp = serializers.DateTimeField(required=True)
+    count = serializers.IntegerField(required=False, allow_null=True)
+    duration = serializers.IntegerField(required=False, default=0)
+    project_id = serializers.IntegerField(required=False, allow_null=True)
+    task_id = serializers.IntegerField(required=False, allow_null=True)
+    job_id = serializers.IntegerField(required=False, allow_null=True)
+    user_id = serializers.IntegerField(required=False, allow_null=True)
```

Comment:
Let's add user_name and user_email. Thus it will be easy to look for a user in ClickHouse and Grafana.